src/main/java/com/solutions/done/VolumeOfCylinder.java

```diff
@@ -0,0 +1,34 @@
+package com.solutions.done;
+
+import java.util.Scanner;
+
+public class VolumeOfCylinder {
+    static double $radius, $height;
+    float $pi;
+    float $voc;
+
+    public VolumeOfCylinder() {
+        $pi = 3.14F;
+    }
+    private void radius(double radius){
```

Comment:
The method name should be in a verb format. For example, in this case: getRadius()